Formula/shogun.rb

```diff
@@ -65,79 +38,68 @@ class Shogun < Formula
   def install
     ENV.cxx11
 
-    # Fix build of modular interfaces with SWIG 3.0.5 on macOS
-    # https://github.com/shogun-toolbox/shogun/pull/2694
-    # https://github.com/shogun-toolbox/shogun/commit/fef8937d215db7
-    ENV.append_to_cflags "-D__ASSERT_MACROS_DEFINE_VERSIONS_WITHOUT_UNDERSCORES=0"
-
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    resource("numpy").stage do
-      system "python", *Language::Python.setup_install_args(libexec/"vendor")
-    end
-
     if MacOS.version >= :high_sierra
       ENV["LAPACKE_PATH"] = Formula["lapack"].opt_lib
     end
 
-    libexec.install resource("jblas")
-
-    python_executable = Formula["python@2"].opt_bin/"python2"
-    python_prefix = Utils.popen_read("#{python_executable} -c 'import sys; print(sys.prefix)'").chomp
-    python_include = Utils.popen_read("#{python_executable} -c 'from distutils import sysconfig; print(sysconfig.get_python_inc(True))'").chomp
-    python_library = "#{python_prefix}/Python"
+    args = std_cmake_args + %w[
+      -DBUILD_EXAMPLES=OFF
```

Comment:
```suggestion
      -DBLA_VENDOR=OpenBLAS
      -DBUILD_EXAMPLES=OFF
```

Reply:
There is an audit that will check packages for linkage against Apple's Accelerate now, so without this change you may trigger an audit warning depending on which version of Homebrew/brew is deployed on our CI infra right now. And if no audit issue is raised now, one should be in the future.

From https://github.com/shogun-toolbox/shogun/issues/4629#issue-442781594
```
-- Found BLAS: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Accelerate.framework  
-- Found Accelerate.framework using as BLAS/LAPACK backend.
-- Enabling Accelerate.framework as BLAS backend for Eigen.
```

Eigen should now be using OpenBLAS as a backend, but since it's a header only lib it just links against whatever implementation the build system provides, if I understand correctly.